docs/fast_data/connectors/debezium_cdc.md

```diff
@@ -8,16 +8,16 @@ This page provides instructions on how to configure the Debezium Server to conne
 
 ## Overview
 
-As previously introduced, in the Mia-Platform Marketplace are available a set of plugins that repackages Debezium Server image, adding the JMX Prometheus Exporter to expose collected metrics regarding the connector.
+As previously introduced, in the Mia-Platform Marketplace are available a set of plugins that repackages Debezium Server image, adding the [JMX Prometheus Exporter](https://github.com/prometheus/jmx_exporter) to expose collected metrics regarding the connector.
```

Comment:
Should we set the nofollow for crawlers on these links? @akelity

Reply:
Should we open an issue in this repo, so that we can keep track of the SEO outside this PR?

Reply:
Sorry for the delay in the response, it's something we have to manage, let's open the issue and proceed from there.